apps/flair/config/integration.exs

```diff
@@ -18,7 +17,7 @@ config :flair,
   divo_wait: [dwell: 1000, max_tries: 60],
   elsa_brokers: endpoint,
   message_processing_cadence: 5_000,
-  data_topic: "persisted",
+  data_topic: "streaming-persisted",
```

Comment:
is this a meaningful distinction to draw?

Reply:
:fire:

Reply:
Forklift specifies `streaming-persisted` in its `config/integration.exs`. I could have changed that value to `persisted`, but I was try to touch as few apps as possible in this PR.

If everyone would prefer the integration/e2e tests use `persisted` or some other value, we can make that change. It doesn't matter IMO.

Reply:
i don't really care either beyond not wanting to propagate the perception from outsiders that it's a meaningful name.

Reply:
I get that. Maybe we change both values to `test-profiling` or something in the future?